src/notation/view/notationviewinputcontroller.h

```diff
@@ -22,6 +22,8 @@
 #ifndef MU_NOTATION_NOTATIONVIEWINPUTCONTROLLER_H
 #define MU_NOTATION_NOTATIONVIEWINPUTCONTROLLER_H
 
+#include <QEvent>
```

Comment:
Oh, sorry, I missed something here: this should be `<QtEvents>` (≠ `<QEvent>`). `<QEvent>` is an alias for `"qcoreevent.h"`, which won't help here. We need `<QtEvents>` instead, which is an alias for `"QtGui/events.h". That contains the definitions of the events we need.

Reply:
done ;-)